editor.planx.uk/src/pages/FlowEditor/lib/__tests__/ordering.test.ts

```diff
@@ -1,173 +0,0 @@
-import { ComponentType as TYPES } from "@opensystemslab/planx-core/types";
```

Comment:
This outdated test suite is removed in favor of new/updated files: 
- `preview/upcomingCardIds.test.ts`
- `preview/autoAnswerableOptions.test.ts`